website/libraries/ui/style.css

```diff
@@ -149,10 +162,242 @@ Legend stuff
     padding: 10px;
 }
 
+.search-button-container, .open-button-container{
+    display: none;
+}
+
+.open-button-container{
+    position: fixed;
+    z-index: 2;
+    bottom: 60px;
+    right: 10px;
+    /* z-index: 5001;
+    bottom: 10px;
+    right: 10px; */
+}
+
+.overlay-title{
+    display: flex;
+    width: 100%;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.open-button, .search-button{
+    background-color: #04944c;
+    color: white;
+
+    width: 100px;
+    height: 35px;
+
+    border: 2px;
+    border-radius: 10px;
+    
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    position: relative;
+    overflow: hidden;
+    transition: background 400ms;
+    outline: 0;
+    border: 0;
+    /* border-radius: 0.25rem; */
+    box-shadow: 0 0 0.5rem rgba(0, 0, 0, 0.3);
+    cursor: pointer;
+}
+
+.open-button-text, .search-button-text{
+    font-size: 15px;;
+    padding-left: 5px;
+    font-weight: 550;
+}
+
+.open-icon, .search-icon{
+    filter: invert(100%);
+    height: 15px;
+    margin-right: 0.3rem;
+}
+
+.search-icon{
+    transform: rotate(180deg);
+}
+
+/* Style the ripple element */
+.ripple {
+    position: absolute;
+    border-radius: 50%;
+    transform: scale(0);
+    animation: rippleAnimation 0.6s linear;
+    background-color: rgba(255, 255, 255, 0.7);
+}
+
+/* Add animation for the ripple effect */
+@keyframes rippleAnimation {
+    to {
+        transform: scale(4);
+        opacity: 0;
+    }
+}
+
+.uk-position-center-right{
+    z-index: 2;
+    width: 150px;
+    margin-right: 1%
+}
+
+.uk-card-default{
```

Comment:
Shouldn't this be in a @media query?

Reply:
Best practice wise probably, but it shouldn't make a difference since the code is only triggered on tap and the button is only visible in mobile
Is it breaking something?

Reply:
yes my bad, it was hardcoded with style attribute in the component so I moved it to style.css but I should have created a specific class. I realized now that it breaks the boxes if you view ot from the computer

Reply:
Fixed in commit f2df2737f9c12b68e73457257a416bc64eec11c7